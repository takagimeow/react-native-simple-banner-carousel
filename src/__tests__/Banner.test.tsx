/* eslint-disable @typescript-eslint/unbound-method */
import React from 'react';
import 'react-native';
import { render } from '@testing-library/react-native';

import { Banner } from '..';

describe('Banner', () => {
  it('Snapshot Test', () => {
    const { toJSON } = render(
      <Banner
        id={`banner_id`}
        source={{
          uri: 'https://github.com/takagimeow/react-native-simple-banner-carousel',
        }}
        width={128}
        onPress={() => console.log('hello world')}
      />,
    );
    expect(toJSON()).toMatchSnapshot();
  });
});
