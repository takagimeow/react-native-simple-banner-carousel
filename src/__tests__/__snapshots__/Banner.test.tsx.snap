// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner Snapshot Test 1`] = `
<View>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    nativeID="animatedComponent"
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "https://github.com/takagimeow/react-native-simple-banner-carousel",
        }
      }
      style={
        Object {
          "backgroundColor": "#bbb",
          "borderRadius": 10,
          "height": 156,
          "maxWidth": "100%",
          "width": 128,
        }
      }
    />
  </View>
</View>
`;
